update.bash

```diff
@@ -6,15 +6,20 @@ if [[ "$(id -u)" -ne 0 ]]; then
   echo "ERROR: You need to be ROOT (sudo can be used)."
   exit 1
 fi
-if ! [[ -f /usr/local/sbin/zram-config ]]; then
+if ! [[ -f /usr/local/sbin/zram-config || -f /usr/sbin/zram-config ]]; then
   echo -e "ERROR: zram-config is not installed.\\nPlease run \"sudo ${BASEDIR}/install.bash\" to install zram-config instead."
   exit 1
 fi
 
-if ! dpkg -s 'gcc' 'make' 'libc6-dev' &> /dev/null; then
+if ! dpkg -s 'gcc' 'make' 'libc6-dev' &> /dev/null && ! [[ "$(grep -o '^ID=.*$' /etc/os-release | cut -d'=' -f2)" == "alpine" ]]; then
   echo "Installing needed packages (gcc, make, libc6-dev)"
   apt-get install --yes gcc make libc6-dev || exit 1
 fi
+if ! [[ "$(apk info 2> /dev/null | grep -E '^(gcc|make|fts-dev)' | tr '\n' ' ')" == "gcc make fts-dev " ]]; then
+  echo "Installing needed packages (gcc, make, fts-dev)"
+  apk add gcc make fts-dev || exit 1
+fi
```

Comment:
Same issue as in install.bash probably always throws errors on Debian causing and early exit.